client/src/components/ProTx/components/dashboard/DisplaySelectors.js

```diff
@@ -9,12 +9,67 @@ import {
 } from '../meta';
 import './DisplaySelectors.module.scss';
 
+const compareSimplifiedValueType = (observedFeature, valueType) => {
```

Comment:
This method *could* (but just a suggestion) move into utils.js in case that check is useful in other components (I could see how it might be useful in plot rendering at some point)

Reply:
Yes, good idea.